src/libraries/System.Text.Json/gen/Reflection/CustomAttributeDataWrapper.cs

```diff
@@ -15,7 +15,7 @@ public CustomAttributeDataWrapper(AttributeData a, MetadataLoadContextInternal m
             var namedArguments = new List<CustomAttributeNamedArgument>();
             foreach (KeyValuePair<string, TypedConstant> na in a.NamedArguments)
             {
-                var member = a.AttributeClass!.GetMembers(na.Key).First();
+                var member = a.AttributeClass.BaseTypes().SelectMany(t => t.GetMembers(na.Key)).First();
```

Comment:
Ideally `a.AttributeClass.BaseTypes()` should be saved in a local above, but that can be included in a follow-up.